cli/tests/pcluster/configure/test_pcluster_configure/test_with_input_no_automation_no_errors_with_config_file/pcluster.config.ini

```diff
@@ -6,11 +6,12 @@ key_name = key2
 base_os = ubuntu1604
 vpc_settings = default
 scheduler = slurm
-master_instance_type = c5.xlarge
-compute_instance_type = g3.8xlarge
+master_instance_type = m6g.xlarge
+compute_instance_type = m6g.xlarge
 max_queue_size = 18
 initial_queue_size = 7
 maintain_initial_size = true
+additional_iam_policies = arn:aws-cn:iam::aws:policy/CloudWatchAgentServerPolicy
```

Comment:
Why adding a conditional policy?

Reply:
It's the region change bug described here: https://github.com/aws/aws-parallelcluster/pull/1915#discussion_r463367524